packages/snowpack/src/rollup-plugins/rollup-plugin-node-process-polyfill.ts

```diff
@@ -0,0 +1,50 @@
+import inject from '@rollup/plugin-inject';
+import {Plugin} from 'rollup';
+import {EnvVarReplacements} from '../types/snowpack';
+
+export function rollupPluginNodeProcessPolyfill(vars: EnvVarReplacements = {}): Plugin {
+  const injectPlugin = inject({
+    process: 'process',
+  });
+
+  return {
+    ...injectPlugin,
+    name: 'snowpack:rollup-plugin-node-process-polyfill',
+    resolveId(source) {
+      if (source === 'process') {
+        return 'process';
+      }
+
+      return null;
+    },
+    load(id) {
+      if (id === 'process') {
+        return createProcessPolyfill(vars);
+      }
+
+      return null;
+    },
+  };
+}
+
+function createProcessPolyfill(vars: EnvVarReplacements = {}) {
+  const env = Object.keys(vars).reduce((acc, id) => {
+    return {
+      ...acc,
+      [id]: vars[id] === true ? process.env[id] : vars[id],
```

Comment:
I inherited this bit from the original `getRollupReplaceKeys` function. I am not sure if the handling of `true` is wise. This could lead to unexpected scenarios in different environments with potentially dire consequences:

Imagine you add a `SUPER_SECRET` env var with the value `true` in your snowpack config. You really just want this to be added as `true` to your environment. Locally this works fine. But on your CI/CD environment, you actually have a SUPER_SECRET env variable set and that now leaks into your build without you even noticing (because it's still truthy!). Whoops!

Reply:
./cc @FredKSchott 

I'd suggest to remove this entirely and just straight up forward whatever is passed as `env` parameters. If someone wants to inherit a env variable, they can just do that manually (by explicitly passing `process.env.FOO`)